main.js

```diff
@@ -1,23 +1,44 @@
 /* VARIABLES   */
 let books = [];
-const renderBooks = document.getElementById('render-books');
+const renderBooks = document.getElementById('render-content');
 const titleInput = document.querySelector('#title');
 const authorInput = document.querySelector('#author');
 const addBook = document.querySelector('#addBook');
 const localBooks = localStorage.getItem('booksStore');
 
+/* CLASS BOOK */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static addItem(title, author) {
+    if (title !== '' && author !== '') {
+      const newBook = new Book(title, author);
+      books.push(newBook);
+    } else {
+      alert('Please fill out the fields in the book form');
+    }
+  }
+
+  static deleteItem(deleteName) {
+    books = books.filter((book) => book.title !== deleteName);
+  }
```

Comment:
- [ ] Take a look at the image below. In the first image I added 3 books with the same title. By the second image, when I delete only one of the three books all three books with the same names are deleted. Kindly fix this issue. I would suggest adding some kind of unique **`id`** for each book and implementing your deletion of books using the unique **`id`** instead of title.

  ![nuri-bug](https://user-images.githubusercontent.com/36542986/148069883-c597c8b2-a5d0-405f-812e-42f1029a263a.jpg)